testgrid/config.yaml

```diff
@@ -51,14 +51,6 @@ default_dashboard_tab:
 #
 
 test_groups:
```

Comment:
Can we have something somewhere to help people understand how all this magic works?

Reply:
Perhaps a generated description that links back to the file that resulted in this being generated?

Reply:
I can, though I kinda doubt anyone will see it given how absurdly large the file is.

Reply:
@spiffxp This file isn't generated - the ones remaining here were hand-crafted, and have some special property that prevents us from automatically generating them. Specifying a testgroup here prevents us from automatically creating a new one.

Reply:
The descriptions show up in testgrid's UI, which might be a more discoverable way of understanding how/why the tabs still exist

Reply:
We aren't autogenerating tabs (those are still done manually), so we can't autogenerate descriptions for them.

(We _will_ add a tab if you specify a `testgrid-dashboards` annotation, in which case the default description does explain why it exists.)

Reply:
To clarify: this only adds _test groups_, which in the majority of cases are regurgitating the same two lines for every job solely to pacify the configurator.

In my experience, the cognitive load of actually figuring out what the two lines should be is remarkably high, and manually writing them does not provide any real benefit.

This PR makes no changes to how dashboards and tabs are specified.

Reply:
(also, I added a comment explaining.)